src/tools/weave/Cmd_GenCodeSigningCert.cpp

```diff
@@ -67,6 +68,11 @@ static const char *const gCmdOptionHelp =
     "       File containing the public and private keys for the new certificate.\n"
     "       (File must be in PEM format).\n"
     "\n"
+    "   -p, --pubkey <file>\n"
+    "\n"
+    "       File containing the public key for the new certificate.\n"
+    "       (File must be in PEM or DER format).\n"
+    "\n"
```

Comment:
Maybe mention that only one of --key or --pubkey can be specified?

Reply:
Will do.